src/lib/utils/options/options.ts

```diff
@@ -81,7 +81,22 @@ export class Options extends ChildableComponent<Application, OptionsComponent> {
     }
 
     read(data: any = {}, mode: OptionsReadMode = OptionsReadMode.Fetch): OptionsReadResult {
-        const event  = new DiscoverEvent(DiscoverEvent.DISCOVER, mode);
+        if (typeof data.extends === 'string') {
+            const tryLoad = (name: string) => {
+                try {
+                    data = {
+                        ...data,
+                        ...require(name)
+                    };
+                    return true;
+                } catch (e) {
+                    return false;
+                }
+            };
+            if (data.extends.startsWith('@') && !data.extends.split('/')[1]) { tryLoad(`${data.extends}/typedoc-config`); } else { tryLoad(`typedoc-config-${data.extends}`) || tryLoad(data.extends); }
```

Comment:
I'm not a huge fan of the many different options here. I think it over complicates the solution. It would be better to just tell the user to specify a file name and error if that file does not exist or fails to load.

We should also probably check for circularity like TypeScript does - http://www.typescriptlang.org/v2/en/tsconfig#extends

Reply:
Fixed!